tests/infer/test_abstract_infer.py

```diff
@@ -5,10 +5,10 @@
 import pyro.distributions as dist
 import pyro.optim as optim
 import pyro.poutine as poutine
-from pyro.infer.autoguide import AutoDelta, AutoDiagonalNormal, AutoLaplaceApproximation
-from pyro.infer import SVI, TracePredictive, Trace_ELBO
```

Comment:
TracePredictive has been deprecated for the last two releases. All the functionality exists in `Predictive`, and these tests are covered there. As such, I thought it safe to remove them since we don't intend to make any changes to `TracePredictive` but instead remove it completely in the next release.